lib/rules/at-rule-empty-line-before/__tests__/index.js

```diff
@@ -278,6 +278,16 @@ testRule({
 			code: '/* foo */\r\n\r\n@media {}',
 			description: 'CRLF',
 		},
+		{
+			code: '// foo\n// bar\n@media {}',
+		},
+		{
+			code: '// foo\n// bar\n\n@media{}',
```

Comment:
I’m expecting this one to fail, given the failing parameters described in GH-4917.